commercetools-internal-processors/src/main/java/io/sphere/sdk/annotations/processors/generators/DraftBuilderGenerator.java

```diff
@@ -128,6 +130,117 @@ private void createCopyFactoryMethods(final TypeElement resourceDraftValueType,
         }
     }
 
+    private boolean isListProperty(PropertyGenModel propertyGenModel){
```

Comment:
The `final` modifier is missing on the parameter.

Reply:
good catch 👍